web-api/src/business/useCases/trialSessions/getPublicTrialSessionDetailsInteractor.ts

```diff
@@ -21,7 +17,6 @@ export type PublicTrialSessionDetails = Pick<
   | 'postalCode'
 > & {
   calendaredCases: RawPublicCase[];
-  isSwingSession: boolean;
```

Comment:
This was a helper that can just as well be calculated on the client-side to avoid passing more data than necessary over the network.